registry/storage/linkedblobstore.go

```diff
@@ -23,7 +23,7 @@ type linkPathFunc func(name string, dgst digest.Digest) (string, error)
 // that grant access to the global blob store.
 type linkedBlobStore struct {
 	*blobStore
-	registry               *registry
+	registry               distribution.Namespace
```

Comment:
I though miminar did it in a PR which he never submitted.  I'll update with master to check.